engine/admin/pruning.md

```diff
@@ -0,0 +1,187 @@
+---
+description: Pruning unused objects
+keywords: pruning, prune, images, volumes, containers, networks, disk, administration
```

Comment:
We should have "garbage collection", "garbage collection" as keyword.

Perhaps even use it in the title ("Garbage collecting - pruning unused Docker objects")